src/UKHO.ERPFacade.Common/Configuration/SapConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace UKHO.ERPFacade.Common.Configuration
+{
+    [ExcludeFromCodeCoverage]
+    public class SapConfiguration : ISapConfiguration
+    {
+        public string BaseAddress { get; set; }
+
+        public string SapServiceOperation { get; set; }
+
+        public string Username { get; set; }
```

Comment:
```suggestion
        public string UserName { get; set; }
```

Reply:
This is added as per keyvault configuration for SAP.